### PR TITLE
[docs] Clarify the use of the approvalMode parameter

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Clarify the use of the approvalMode parameter.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: Clarify the use of the approvalMode parameter.
impact_level: low
```

### CODE REVIEW

File: candi/openapi/doc-ru-node_group.yaml [hunk 1]
```diff
@@ -156,9 +156,11 @@ spec:
...
+                        - `Manual` — отключить автоматическую выдачу разрешений на disruptive-обновление. Если потребуется disruptive-обновление, то загорится специальный алерт. **Внимание!** Для группы узлов `master` режим выдачи разрешения всегда должен быть `Manual`, чтобы избежать проблем при drain'е узла.
                         - `Automatic` — автоматически выдавать разрешения на disruptive-обновление.
                         - `RollingUpdate` — в этом режиме будет создан **новый** узел с обновленными настройками, а старый узел будет удален. Разрешено только для облачных узлов.
+
+                        Если не используется режим `RollingUpdate`, то при обновлении с узла выгоняется нагрузка (drain), узел обновляется (перезагружается) и вводится в работу. При таком режиме, в кластере должно быть место для размещения нагрузки на время недоступности обновляемого узла. В режиме `RollingUpdate` происходит **замена** узла на обновленный, т.е. на время обновления в кластере появляется дополнительный узел. В облачной инфраструктуре режим `RollingUpdate` удобен, например, если в кластере нет ресурсов для временного размещения нагрузки с обновляемого узла.
```
Comment (kirkonru) [inline candi/openapi/doc-ru-node_group.yaml:163]:
```suggestion
                        Если не используется режим `RollingUpdate`, при обновлении узел освобождается от нагрузки (drain), после чего он обновляется (перезагружается) и вводится в работу. Обратите внимание, что в этом случае в кластере должно быть место для размещения нагрузки на время, пока обновляемый узел недоступен. В режиме `RollingUpdate` узел **заменяется** на обновленный, т.е. на время обновления в кластере появляется дополнительный узел. В облачной инфраструктуре режим `RollingUpdate` удобен, например, если в кластере нет ресурсов для временного размещения нагрузки с обновляемого узла.
```

File: candi/openapi/node_group.yaml [hunk 2]
```diff
@@ -461,6 +461,8 @@ spec:
                         - `Manual` — disable automatic disruption approval; the alert will be displayed if disruption is needed. **Caution!** The `master` node group update mode must be `Manual` to avoid issues with draining.
                         - `Automatic` —  automatically approve disruption-involving updates.
                         - `RollingUpdate` — in this mode, a **new** node with new settings will be created; then, the old node will be deleted. Available only for cloud nodes.
+
+                        If the `RollingUpdate` mode is not used, then first the node is drained. After that the node updates (reboots) and uncordons. In this mode, there should be resources in the cluster to run the load during the unavailability of the updated node. In the `RollingUpdate` mode the node is **replaced** with an updated one, i.e., an additional node appears in the cluster during the update. In the cloud infrastructure, the `RollingUpdate` mode is convenient, for example, if the cluster has no resources to temporarily run the load from the node being updated.
```
Comment (kirkonru) [inline candi/openapi/node_group.yaml:465]:
```suggestion
                        If the `RollingUpdate` mode is not used, when updating, the node is first drained and then updated (rebooted) and put back into operation (uncordoned). Note that in this case, the cluster must have sufficient resources to accommodate the load while the node being updated is unavailable. In the `RollingUpdate` mode, the node is **replaced** by the updated node, i.e., an extra node appears in the cluster for the duration of the update. In cloud infrastructures, the `RollingUpdate` mode is convenient, for example, if there are no resources in the cluster to temporarily host the load from the node being updated.
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4092](https://github.com/deckhouse/deckhouse/pull/4092) to the branch [release-1.44](https://github.com/deckhouse/deckhouse/tree/release-1.44) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4167](https://github.com/deckhouse/deckhouse/pull/4167) to the branch [release-1.43](https://github.com/deckhouse/deckhouse/tree/release-1.43) successful!

**Pull request merged**